### PR TITLE
[ISV-1053] Check whether certProject.operator_distribution value matches with config of submission repo

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

**Comment by haripate** [comment]
I have placed this condition in `get-cert-project-related-data.yml` because it is easy and efficient to get the value of `certProject.operator_distribution` here.

cc @amisstea

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml [hunk 1]
```diff
@@ -46,6 +48,27 @@ spec:
           --pyxis-url $(params.pyxis_url) \
           --cert-project-id $(params.cert_project_id)
 
+        ORGANIZATION=$(cat config.yaml | yq -r '.organization')
+        OPERATOR_DISTRIBUTION=$(cat cert_project | jq '.operator_distribution'| tr -d '"')
+        case "$ORGANIZATION" in
+          "certified-operators")
+            if [ "$OPERATOR_DISTRIBUTION" != "connect" ]; then
+              echo "The operator distribution $OPERATOR_DISTRIBUTION does not belong to $ORGANIZATION organization"
```
Comment (amisstea) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml:56]: Please change this message (and the similar one below) to:
```
This project is not permitted to distribute to the $ORGANIZATION catalog.
```

**Review by amisstea — COMMENTED** [review]
Everything else looks good. :+1:

File: ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml (hunk 1 above)
Comment (haripate) [reply ansible/roles/operator-pipeline/templates/openshift/tasks/get-cert-project-related-data.yml]: Updated!

**Review by amisstea — APPROVED** [review]

**Pull request merged**